packages/build/src/error/types.ts

```diff
@@ -39,10 +113,44 @@ export const getTypeInfo = function ({ type }) {
 //    "All errors" bookmark depending on whether we should get notified on Slack
 //    for new errors of that type. You must use the bookmark menu action "Update
 //    with current filters"
-const TYPES = {
+export interface ErrorType {
+  title: TitleFunction | string
+  locationType?: string
+  showErrorProps?: boolean
+  rawStack?: boolean
+  showInBuildLog?: boolean
```

Comment:
nit: I know this was already in place before your PR, but it'd be nice if we could move the comments above to JSDoc comments next to each property, so that they're picked up by Intellisense.

Reply:
Done via - https://github.com/netlify/build/pull/5374/commits/284f4887d01ffe1f08d1b5c06e9dbb2b27c35265